sklearn/decomposition/fastica_.py

```diff
@@ -431,7 +348,10 @@ def my_g(x):
     Attributes
     ----------
     components_ : 2D array, shape (n_components, n_features)
-        The unmixing matrix.
+        The linear operator to apply to the data to get the independent
+        sources. This is equal to the unmixing matrix when ``whiten`` is
+        False, and equal to ``np.dot(unmixing_matrix, self.whitening_)`` when
+        ``whiten`` is True.
 
     mixing_ : array, shape (n_features, n_components)
```

Comment:
you're going to hate me @NicolasHug but now mixing_ is not the inverse of unmixing in your description. unmixing is always square where here mixing_ is not.

also why having mixing_ as public attribute and not unmixing_ ?

Reply:
How would you fix it? Change the docstring?

This issue is not caused by the changes: `mixing_` has never been the inverse of the unmixing matrix.